CHANGELOG.rst

```diff
@@ -12,12 +12,18 @@ This document describes changes between each past release.
 - The ``cli_utils.set_parser_server_options()`` was renamed
   ``cli_utils.add_parser_options()`` (#63)
 
+
 **New features**
 
 - The ``cli_utils`` can now handle option bucket and collection
   parameters. (#63)
```

Comment:
This is a bit lacking, maybe something along the lines of:
`... can now exclude bucket and collection parameters`